src/lib/ThumbnailsSidebar.js

```diff
@@ -9,6 +9,7 @@ const CLASS_BOX_PREVIEW_THUMBNAIL_IMAGE_LOADED = 'bp-thumbnail-image-loaded';
 const CLASS_BOX_PREVIEW_THUMBNAIL_IS_SELECTED = 'bp-thumbnail-is-selected';
 const CLASS_BOX_PREVIEW_THUMBNAIL_PAGE_NUMBER = 'bp-thumbnail-page-number';
 export const DEFAULT_THUMBNAILS_SIDEBAR_WIDTH = 154; // 225px sidebar width - 25px margin right, - 40px for page number - 6px for border
+const HIGHER_RES_WIDTH = DEFAULT_THUMBNAILS_SIDEBAR_WIDTH * 4;
```

Comment:
I don't usually ask this, but can we add a comment or move `4` to a constant to indicate that it's an upscale multiplier? The purpose of it may get lost with time, otherwise.